Amplify/Categories/Auth/Models/DeliveryDestination.swift

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+public typealias Destination = String
+
+public enum DeliveryDestination {
+
+    case email(Destination?)
+    case sms(Destination?)
+    case phone(Destination?)
```

Comment:
nit: For initial releases of enums that don't have `Int` raw values, alphabetize the non-default, non-unknown cases.